test/utilities/DominoDocumentFragment.test.js

```diff
@@ -0,0 +1,29 @@
+import assert from 'assert'
+import domino from 'domino'
+import dominoDocumentFragment from './DominoDocumentFragment'
+
+const documentFragmentFromHTMLString = dominoDocumentFragment.documentFragmentFromHTMLString
+
+describe('DominoDocumentFragment', () => {
+  describe('documentFragmentFromHTMLString()', () => {
+    it('should return a Domino DocumentFragment when given an HTML string', () => {
+      const frag =
+        documentFragmentFromHTMLString('<div>Oh look! A fragment!</div><a id="link1"></a>')
+      assert.ok(frag instanceof domino.impl.DocumentFragment)
+    })
+    it('should return a Domino DocumentFragment with expected childNodes', () => {
+      const frag =
+        documentFragmentFromHTMLString('<div>Oh look! A fragment!</div><a id="link1"></a>')
```

Comment:
Consider replacing globally replacing `<div>Oh look! A fragment!</div>` with `<div>text</div>` to fit this statement on one line.